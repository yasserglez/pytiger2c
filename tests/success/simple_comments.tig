/* This is a Tiger program that have comments almost everywhere. */

let 
    /* This is a comment inside the declaration part. */
    var i := 42 /* A new n integer variable is declared. */
    
    /* This is a comment let that should be ignored.
     * var j := 1
     */
     
     var k := 1
     
    /* end */
in
    /**************************************************
     * This is a comment block inside the code part.  *
     **************************************************/
    
    print("What's the value of i here? "); /* A comment between prints. */ printi(i);
    /* Do not print this: print(i); */ /* print(j); */
    printi /* Ignore this */ ( /* this */ i /* this */ ) /* and, of course, this. */ ;
    print("\n")
end

/* This is a comment at the end of the file. */
