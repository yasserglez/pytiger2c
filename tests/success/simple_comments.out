What's the value of i here? 4242
